front/src/modules/ui/board/components/__stories__/BoardColumnEditTitleMenu.story.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { getRenderWrapperForComponent } from '~/testing/renderWrappers';
+
+import { BoardColumnEditTitleMenu } from '../BoardColumnEditTitleMenu';
+
+const meta: Meta<typeof BoardColumnEditTitleMenu> = {
+  title: 'UI/Board/BoardColumnMenu',
+  component: BoardColumnEditTitleMenu,
+};
+
+export default meta;
+type Story = StoryObj<typeof BoardColumnEditTitleMenu>;
+
+export const AllTags: Story = {
+  render: getRenderWrapperForComponent(
+    <BoardColumnEditTitleMenu
+      color="green"
+      title={'Column title'}
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
I think those should be optional on BoardColumnEditTitleMenu, if they are not provided as props, they won't be called and that's it